scripts/biomedical/diseaseOntology/disease_ontology.tmcf

```diff
@@ -0,0 +1,24 @@
+Node: E:DiseaseOntology->E1
+typeOf: dcs:Disease
```

Comment:
Update script to include a column of text values with the disease ontology ids eg "DOID:0060329" then add to the tmcf the line `diseaseOntologyID: C:DiseaseOntology->diseaseOntologyID`